docs/03-User Guides/01-Pipeline Programing Guide/01-node-types.md

```diff
@@ -8,12 +8,12 @@ Currently, Towhee supports nine types of nodes. Input and output nodes are used
+| **output**(*output_schema*)                                  | This node defines the pipeline's output schema, and ends a pipeline definition. Once called, a pipeline instance will be created and returned. Refer to [output API](/05-API%20Reference/01-Pipeline%20API/02-output.md) for more details. |
 | **map**(*input_schema, output_schema, func*)                 | This node applies the given function `func` to each of its inputs and returns the transformed data. `map` returns one row for every row of input. Refer to [map API](/05-API%20Reference/01-Pipeline%20API/03-map.md) for more details. |
-| **flat_map***(**input_schema, output_schema,* *func)*        | This node flattens the results after applying the function to every row of input, and returns the flattened data respectively.The returned data can have the same count or more number of rows compared with the input. This is one of the major differences between `flat_map` and `map`, where `map` always returns the same number of rows as input. Refer to [flat_map API](/05-API%20Reference/01-Pipeline%20API/04-flat-map.md) for more details. |
-| **filter**(*input_schema, output_schema,* *filter_columns, func*) | This node applies the filter function `func` to the `filter_columns`.Refer to [filter API](/05-API%20Reference/01-Pipeline%20API/05-filter.md) for more details. |
-| **window**(*input_schema, output_schema,* *size, step, func*) | This node batches the input rows into multiple rows based on the specified window `size` and `step`. Then it applies a function `func` to each of the windowed data, and returns the results - one row of results for each of the windows. Refer to [window API](/05-API%20Reference/01-Pipeline%20API/06-window.md) for more details. |
-| **time_window**(*input_schema, output_schema,* *timestamp_col, size, step, func*) | This node is used to batch rows that have a time sequence, for example, audio or video frames.`time_window` is similar to `window`, but the batching rule is applied based on a timestamp column (`timestamp_col`). `size` is the time interval of each window, and `step` determines how long a window moves from the previous one. Note that if `step` is less than `size`, the windows will overlap. Refer to [time_window API](/05-API%20Reference/01-Pipeline%20API/07-time-window.md) for more details. |
+| **flat_map**(**input_schema, output_schema, func*)           | This node flattens the results after applying the function to every row of input, and returns the flattened data respectively.The returned data can have the same count or more number of rows compared with the input. This is one of the major differences between `flat_map` and `map`, where `map` always returns the same number of rows as input. Refer to [flat_map API](/05-API%20Reference/01-Pipeline%20API/04-flat-map.md) for more details. |
```

Comment:
![image](https://github.com/towhee-io/towhee/assets/14136703/4ab1aef2-30a1-42ee-8d09-9d2844202cb9)
The formatting of flat_map still has some issues.